typings/objection/index.d.ts

```diff
@@ -857,6 +861,17 @@ declare namespace Objection {
     (column: ColumnRef, callback: (this: QueryBuilder<T>, queryBuilder: QueryBuilder<T>) => void): QueryBuilder<T>;
   }
 
+  interface FindOne<T> {
```

Comment:
Since findOne should accept anything that where accepts, shouldn't the call signatures be the same with Where<T> with different return value?

Now there seem to be extra methods:

```
(condition: boolean): QueryBuilderOption<T>;
...
(sql: string, ...bindings: any[]): QueryBuilderOption<T>;
(sql: string, bindings: any): QueryBuilderOption<T>;
```

Reply:
Yup, that's the point! [This diff](https://github.com/Vincit/objection.js/commit/559d38df9992a76cbd9213e6e64b8acc0c8cc776) failed to make `.findOne` and `.where` have equivalent argument parameters. Now they match.